docs/changes.rst

```diff
@@ -229,7 +229,9 @@ Other changes:
     to break was fixed in cygwin at least since cygwin 2.8, maybe even since
     2.7.0).
   - remove skipping the noatime tests on GNU/Hurd, #2710
-
+  - fix borg import issue, add comment, #2718
```

Comment:
Actually I don't think that this was released, since it was part of the manifest feature flags changeset.